docs/resources/synthetics.md

```diff
@@ -134,7 +143,7 @@ The following arguments are supported:
     - `targetvalue` - (Required) Expected matching value
     - `jsonpath` - (Required) The JSON path to assert
   - `property` - (Optional) if assertion type is "header", this is a the header name
-- `options` - (Required)
+- `options` - (Required) **Deprecated**
```

Comment:
I think you need to add the deprecation now.

Reply:
Ah right I forgot to add it in the schema